### DISCUSSION TITLE
Lens search for PDF export in the developer guide returns obsolete results

### DISCUSSION THREAD

**Discussion by JaniceManwiller** [GitbookIO/community#319] [5. Export]
We just upgraded to Pro and enabled PDF export. We were curious whether there was an API we could use to automate PDF export of a space.

When I did a search for PDF export on this forum, I came across a post from January 2023 that mentioned that the CLI option to export a PDF had been removed.

A Lens search of the developer guide actually returns information about the CLI option, with a link to the CLI reference. But the CLI reference does not contain the option. So it looks like the information that Lens is using is out of date.

![image](https://github.com/GitbookIO/community/assets/107077736/c48ee338-be7d-4156-af32-2990b2e477aa)

**Comment by aikoii** [comment]
👋 @JaniceManwiller, currently, GitBook does not have a dedicated API endpoint for automating PDF exports. The CLI option was indeed deprecated. I'm sorry for the confusion! We are continuously working to improve our GitBook AI, and I've taken a note of this internally to ensure it's corrected 🙂.
